spec/controllers/users_controller_spec.rb

```diff
@@ -1,4 +1,28 @@
 require 'rails_helper'
 
 RSpec.describe UsersController, type: :controller do
+  describe 'GET #login' do
+    before { fake_cas }
+
+    it 'redirects to root path by default' do
+      get :login
+      expect(response).to redirect_to root_path
+    end
+
+    it 'redirects to the specified url' do
+      url = '/somewhere'
```

Comment:
This actually does work. `redirect_to` constructs a 302 response to wherever you tell it to go, so you can also have it redirect to completely different domains. Should I have it be an actual URL in the project?